orodha-keycloak/tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains two fixtrues which supply data to our KeycloakConnection mock in
+lieu of using the python-keycloak package to connect to our server.
+"""
+import pytest
+import unittest.mock
+from fixtures import MOCKDATA
+
+class MockKeycloakOpenIDConnection(unittest.mock.MagicMock):
```

Comment:
@Edison-Stuart This is actually mocking `KeycloakAdmin`, so we should call it `MockKeycloakAdmin`.